docs: Add clarification about `eslint-enable` (#15680)
eslint/eslint: docs/user-guide/configuring/rules.md

/* eslint-enable no-alert, no-console */
```

> **Note:** `/* esline-enable */` without any specific rules listed will cause all disabled rules to be re-enabled.

Reviewer: ```suggestion
**Note:** `/* eslint-enable */` without any specific rules listed will cause all disabled rules to be re-enabled.
```

Typo

> **Note:** `/* esline-enable */` without any specific rules will cause all errors to be re-enabled. You will have to re-specify the errors you wish to keep disabled.

Reviewer 2: ```suggestion
**Note:** `/* esline-enable */` without any specific rules listed will cause all disabled rules to be re-enabled.
```

alert('foo');
```

To ensure that a rule is always applied (regardless of any future enable/disable lines):

```js
/* eslint no-alert: "off" */

alert('foo');

> ```

Reviewer: Isn't it the opposite, this ensures that a rule is _not_ applied regardless of enable/disable comments?

Author: You're right, I'll go ahead and update that now
